yellowbrick/features/radviz.py

```diff
@@ -138,10 +138,11 @@ class RadialVisualizer(DataVisualizer):
     """
```

Comment:
Awesome job with this PR.  However, a description of "Alpha" must be included in the doc string.  After this, I will merge it